akka-cluster-sharding/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -3,3 +3,16 @@ ProblemFilters.exclude[Problem]("akka.cluster.sharding.Shard.*")
 
 # #25191
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.retryTask")
+
+# #27100 Productionize: GetShardRegionStats returns empty shard set on ask timeout
+# askAllShards, an internal function, was renamed and changed to query all or a subset of shards to try failures only
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.askAllShards")
+# Added new field for failed shard queries to akka.cluster.sharding.ShardRegion#ShardRegionStats, and updated in proto
+ProblemFilters.exclude[MissingTypesProblem]("akka.cluster.sharding.ShardRegion$ShardRegionStats$")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion#ShardRegionStats.apply")
```

Comment:
Is it typical when adding a field to add a backward compat `apply` or just add the filter?

Reply:
You'll need to add a backward compatible apply